### PR TITLE
Elm json fix

### LINKED ISSUE #14: Couldn't decode Elm 0.19 package elm.json
Run elmi-to-json in a folder with the following `elm.json` file:

```json
{
    "type": "package",
    "name": "avh4/elm-fifo",
    "summary": "First in, first out (FIFO) queue",
    "license": "MIT",
    "version": "1.0.4",
    "exposed-modules": [
        "Fifo"
    ],
    "elm-version": "0.19.0 <= v < 0.20.0",
    "dependencies": {
        "elm/core": "1.0.0 <= v < 2.0.0"
    },
    "test-dependencies": {}
}
```

produces the following error: 

```
elmi-to-json failed for:
all

Couldn't decode /Users/avh4/workspace/elmi-to-json/tmp_test_workspace/elm.json
```

### PR DESCRIPTION
**Pull request by stoeffel**
fixes #14

### CODE REVIEW

File: src/Elm/Json.hs [hunk 1]
```diff
@@ -23,7 +23,8 @@ data ElmJson = ElmJson
 instance Aeson.FromJSON ElmJson where
   parseJSON =
     Aeson.withObject "Coord" $ \v ->
-      ElmJson <$> v .: "source-directories" <*> v .: "elm-version"
+      ElmJson <$> v .:? "source-directories" .!= ["src"] <*>
```
Comment (stoeffel) [inline src/Elm/Json.hs]: I think we should add `tests` to the default src dirs

**Comment by avh4** [comment]
Ah, yes, `./tests` needs to be added, though I'm not sure if it's 100% that the new elm-test will always use that folder... I think there was discussion of if it should be "test" instead of "tests"?  And it's possible elm-test might have a config option, but maybe it will just require that tests are always in the tests folder.  Anyway, the way it works now is just what I did to get it working, and I'm not sure if Richard or someone else might revisit that decision later.

**Comment by avh4** [comment]
Oh.. nevermind, what I just said doesn't matter because elm-test always generates an application elm.json in the generated folder that has all the necessary source directories listed.

**Comment by avh4** [comment]
But... I'm not sure if tests should really be added or not.   elm-test only compiles the tests from cwd of the generated folder, and will never compile them from the project root.  Also, tests will probably not compile from the test root anyway because they will need things in the test-dependencies, which elm make at the top level won't include.

However, that all is likely to be a problem for editor plugins that want to show compile errors for tests in 0.19.  I haven't heard any discussion yet from anyone about how that will be dealt with, so maybe that would inform what elmi-to-json should do here.

**Comment by stoeffel** [comment]
I think it doesn't hurt to have it in there since it will first check `src` and then `tests`. So I think I will merge this. okay @avh4?

**Comment by avh4** [comment]
okay

**Pull request merged**